src/PHPStan/Faker/Reflection/FakerPropertiesClassReflectionExtension.php

```diff
@@ -36,7 +37,7 @@ public function __construct()
 			'dateTimeThisDecade' => [new ObjectType(DateTime::class), false, false, true],
 			'dateTimeThisMonth' => [new ObjectType(DateTime::class), false, false, true],
 			'dateTimeThisYear' => [new ObjectType(DateTime::class), false, false, true],
-			'rgbColorAsArray' => [new ArrayType(new IntegerType()), false, false, true],
+			'rgbColorAsArray' => [new ArrayType(new MixedType(), new IntegerType()), false, false, true],
```

Comment:
What exactly does this change mean?

Reply:
For ArrayType class we need to pass 2 required parameters. [ArrayType](https://github.com/phpstan/phpstan/blob/master/src/Type/ArrayType.php)

But today i thought and i think that for rgbColorAsArray as type for array key is more suitable IntegerType. I have changed it.